account_journal_lock_date/__manifest__.py

```diff
@@ -5,11 +5,11 @@
     "name": "Account Journal Lock Date",
     "summary": """
         Lock each journal independently""",
-    "version": "12.0.2.0.0",
+    "version": "13.0.1.0.0",
     "license": "AGPL-3",
-    "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
+    "author": "ACSONE SA/NV, Odoo Community Association (OCA)",
```

Comment:
Shouldn't we be here after the major feature we added?

Reply:
Yes, I think we should. Done